### ISSUE TITLE
num_rand_file_per_scene 

### ISSUE THREAD

**Issue by staymylove** [pengsongyou/openscene#61]
Dear author:
    I want to ask some question in the code:
     if split== 'train': # for training set, export a chunk of point cloud
        args.n_split_points = 20000
        args.num_rand_file_per_scene = 5
    
   Why set the parameter 'num_rand_file_per_scene' =5?
   In the given fused_feartures file.pt , so we can just get and use 5 objects features in one scene?
    
    Looking forward to your reply.

**Comment by pengsongyou** [comment]
So to distill the 3D model, we only take the sparse point clouds in each scene as the pseudo supervision, so we just randomly sample 20K pionts for each training scenes, and get 5 of them. However, for the validation set, we just sample all points :) Not sure if that answers your questions?

**Comment by staymylove** [comment]
emmm，
I want to ask, there are many objects in one scene, why not every object has its corresponding 2d-features, and there are only 5 features pt file in one scene?

**Comment by pengsongyou** [comment]
Ah, no, it is 5 files with features of the entire scene, not each object.

**Comment by staymylove** [comment]
Oh, I see. Thanks for your reply. 
So one scene consists of 5 files, but how to segment the scene to 5 files? Are there some rules or other methods. 
And if I want to use the features of entire scene, what should I do by using the 5 files? Should I concat the 5 files or some other aggregation methods?

**Comment by pengsongyou** [comment]
Just randomly sample points, please check our [code](https://github.com/pengsongyou/openscene/blob/main/scripts/feature_fusion/fusion_util.py#L79). Nope, 5 files are only for training. For evaluation or testing, we sample all points in the scene. If there are 200K points, we simply take all these 200K points without sampling 50K.

**Comment by staymylove** [comment]
Oh I got it. Thanks for your patient reply! It is a wonderful paper and code. Hope to see your more wonderful papers!
